components/cache/cache_pools.rst

```diff
@@ -108,6 +109,46 @@ helper allows creating a connection to a Redis server using a DSN configuration:
 
     $redisConnection = RedisAdapter::createConnection('redis://localhost');
 
+    $cache = new RedisAdapter($redisConnection);
+
+See the method's docblock for more options.
+
+Memcached Cache Adapter
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This adapter stores the contents into a set of `Memcached`_ servers.
+
+Before you start, make sure you have Memcached running and have created a
+:phpclass:`Memcached` object::
+
+    use Symfony\Component\Cache\Adapter\MemcachedAdapter;
+
+    $cache = new MemcachedAdapter(
+        // the object that stores a valid connection to your Memcached servers
+        \Memcached $client,
+        // the string prefixed to the keys of the items stored in this cache
+        $namespace = '',
+        // in seconds; applied to cache items that don't define their own lifetime
+        // 0 means to store the cache items indefinitely (i.e. until the Memcached memory is deleted)
+        $defaultLifetime = 0
+    );
+
+The :method:`Symfony\\Component\\Cache\\Adapter\\Memcached::createConnection`
+helper allows creating a connection to a pool of Memcached server using a DSN configuration::
+
+    $memcachedClient = MemcachedAdapter::createConnection(
+        'memcached://user:pass@localhost?weight=33',
+
+        // options, including username, password and Memcached::OPT_* options
+        array('persistent_id' => '_products_cache')
+    );
+    $memcachedClient = MemcachedAdapter::createConnection(array(
```

Comment:
Is this repeated `$memcachedClient` variable defined on purpose? Should we merge its contents with the previous variable?